app/webpacker/components/Panel/pages/ApprovePictures/AvatarCard.jsx

```diff
@@ -0,0 +1,170 @@
+        centered
+        style={{
+          objectFit: 'contain',
+          height: '250px',
+          width: '250px',
+        }}
+      />
+      <Divider horizontal />
+      <ButtonGroup toggle>
+        {actions.map((action) => (
+          <Button
+            key={action.label}
+            onClick={action.onClick}
+            color={action.color}
+            basic={action.basic}
+            disabled={action.disabled}
+          >
+            {action.label}
+          </Button>
+        ))}
+      </ButtonGroup>
+    </Segment>
+  );
+}
+
+export default function AvatarCard({
+  user, onReject, onApprove, onDefer,
+}) {
+  const [activeAvatarIndex, setActiveAvatarIndex] = useState(0);
+  const [avatarState, setAvatarState] = useState('');
+  const [rejectionReason, setRejectionReason] = useState('');
+
+  const possibleRejectionReasons = useMemo(() => {
+    const r = I18n.tArray('users.edit.avatar_guidelines');
+    if (user['staff_or_any_delegate?']) {
+      return r.concat(I18n.tArray('users.edit.staff_avatar_guidelines.paragraphs'));
+    }
+    return r;
+  }, [user]);
+
+  const currentAvatarActions = useMemo(() => [
+    {
+      label: 'Previous',
+      onClick: () => setActiveAvatarIndex(activeAvatarIndex - 1),
+      color: 'blue',
+      basic: true,
+      disabled: activeAvatarIndex === 0,
+    },
+    {
+      label: 'Next',
+      onClick: () => setActiveAvatarIndex(activeAvatarIndex + 1),
+      color: 'blue',
+      basic: true,
+      disabled: activeAvatarIndex === user.avatar_history.length - 1,
+    },
+  ], [activeAvatarIndex, user.avatar_history.length]);
+
+  const pendingAvatarActions = [
+    {
+      label: 'Approve',
+      onClick: () => {
+        onApprove(user.pending_avatar);
+        setAvatarState('approved');
+      },
+      color: 'green',
+      disabled: avatarState === 'approved',
+    },
+    {
+      label: 'Reject',
+      onClick: () => {
+        if (avatarState === 'chooseRejectionReason') {
+          if (rejectionReason === '') {
+            setAvatarState('');
+          } else {
+            onReject(user.pending_avatar, rejectionReason);
+            setAvatarState('rejected');
+          }
+        } else {
+          setAvatarState('chooseRejectionReason');
+        }
+      },
+      color: 'red',
+      disabled: avatarState === 'rejected',
+    },
+    {
+      label: 'Defer',
+      onClick: () => {
+        onDefer(user.pending_avatar);
+        setAvatarState('deferred');
+      },
+      color: 'grey',
+      disabled: avatarState === 'deferred',
+    },
+  ];
+
+  return (
+    <Card fluid>
+      <CardContent>
+        <CardHeader>
+          <a href={editPersonUrl(user.id)}>
+            {user.name}
+          </a>
+        </CardHeader>
+        { user['staff_or_any_delegate?'] && (
+          <CardMeta>
+            <strong>Staff Member or Trainee Delegate</strong>
+            {' '}
+            - see guidelines above
+          </CardMeta>
+        )}
```

Comment:
Could we only show the guidelines when the user is a staff, instead of always showing them and conditionally showing this message?

Reply:
not sure what you mean

Reply:
`- see guidelines above` is referring to the items below `<Header as="h4">Additional guidelines for Staff Members</Header>` in the index file. I'm suggesting only show that header and those items if the user is a staff/delegate, instead of always showing the guidelines and sometimes showing this warning.

Reply:
Well the problem with that is that there might be 10 request, but only one staff. The index page would need to determine if any of the ten people are staff to display the additional guidelines, which I think will be quite jarring of an UI update once that one staff avatar request is worked off